### PR TITLE
Update fractional zoom

### PR DESCRIPTION
**Pull request by saidaipparla**

### CODE REVIEW

**Comment by mbarto** [comment]
Please solve the conflicts so that I can merge

File: web/client/components/map/leaflet/__tests__/Map-test.jsx [hunk 1]
```diff
@@ -169,7 +169,7 @@ describe('LeafletMap', () => {
 
         const leafletMap = map.map;
         expect(leafletMap.getZoom()).toBe(12);
-        map.setProps({zoom: 12, center: {y: 44, x: 10}});
+        map.setProps({zoom: 12.4, center: {y: 44, x: 10}});
         expect(leafletMap.getZoom()).toBe(12);
```
Comment (Gnafu) [inline web/client/components/map/leaflet/__tests__/Map-test.jsx]: This test is to "check if the map changes when receive new props"
The zoom is 12 before the test, so you should expect another value after the setProps().

File: web/client/components/map/openlayers/__tests__/Map-test.jsx [hunk 2]
```diff
@@ -168,7 +168,7 @@ describe('OpenlayersMap', () => {
 
         const olMap = map.map;
         expect(olMap.getView().getZoom()).toBe(12);
-        map.setProps({zoom: 12, center: {y: 44, x: 10}});
+        map.setProps({zoom: 12.4, center: {y: 44, x: 10}});
         expect(olMap.getView().getZoom()).toBe(12);
```
Comment (Gnafu) [inline web/client/components/map/openlayers/__tests__/Map-test.jsx]: Same as before: here the test should expect a different value than before the setProps()

File: web/client/components/map/leaflet/__tests__/Map-test.jsx (hunk 1 above)
Comment (saidaipparla) [inline web/client/components/map/leaflet/__tests__/Map-test.jsx]: ok i will change

**Pull request merged**